tests/parser-tests.lisp

```diff
@@ -44,4 +44,6 @@
       (parse-file file))))
 
 (deftest test-parse-package-suite ()
+  (let ((*features* (cons ':coalton-lisp-toplevel *features*)))
```

Comment:
can we bind this in the `.asd` like the other one?

Reply:
I just tried, and, it has no effect, because the feature is checked during test execution.